src/core/cli_arg.cpp

```diff
@@ -58,8 +59,14 @@ bool CliArg::find_protocol(std::string& rest)
         return true;
     } else if (rest.find(serial + delimiter) == 0) {
         _protocol = Protocol::Serial;
+        _flow_control = false;
         rest.erase(0, serial.length() + delimiter.length());
         return true;
+    } else if (rest.find(serial_flowcontrol + delimiter) == 0) {
+        _protocol = Protocol::Serial;
+        _flow_control = true;
+        rest.erase(0, serial_flowcontrol.length() + delimiter.length());
+        return true;
```

Comment:
I should have told you earlier, there is a test for this:
https://github.com/mavlink/MAVSDK/blob/develop/src/core/cli_arg_test.cpp

Would be great if you could add coverage for this.

Reply:
Of course, I should have checked. Let's see if they pass.